pyproject.toml

```
@@ -1,3 +1,76 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "paho-mqtt"
+dynamic = ["version"]
+description = "MQTT version 5.0/3.1.1 client class"
+readme = "README.rst"
+# see https://lists.spdx.org/g/Spdx-legal/topic/request_for_adding_eclipse/67981884
+# for why Eclipse Distribution License v1.0 is listed as BSD-3-Clause
+license = "EPL-2.0 OR BSD-3-Clause"
+requires-python = ">=3.7"
+authors = [
+    { name = "Roger Light", email = "roger@atchoo.org" },
+]
+keywords = [
+    "paho",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved",
+    "Natural Language :: English",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Communications",
+    "Topic :: Internet",
+]
+dependencies = []
+
+[project.optional-dependencies]
+proxy = [
+    "PySocks",
+]
+
+[project.urls]
+Homepage = "http://eclipse.org/paho"
+
+[tool.hatch.version]
+path = "src/paho/mqtt/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+    "src/paho",
+    "/examples",
+    "/tests",
+    "about.html",
+    "CONTRIBUTING.md",
+    "edl-v10",
+    "epl-v10",
```

```suggestion
    "epl-v20",
```

To match change done in #635